### DISCUSSION TITLE
Infinite loop of comments when coming from notifications

### DISCUSSION THREAD

**Discussion by albertlarsan68** [community/community#43361] [Mobile]
### Select Topic Area

Bug

### Body

Hello,

Sometimes, on mobile, when I review my notifications, I get "caught" in an infinite loop.
Let me explain more:
- I open a notification on GitHub mobile
- I scroll down the PR, and after some time I start to loop around the last comments, and label changes, etc., they repeat infinitely.

I will give links to those kind of notification when I can.

**Comment by HenSquared** [comment]
Hi! thanks for reporting this issue.

What device are you using? 

Can you share a screen recording or screenshots of this behavior? This will help us get a better idea of what is happening. 

Thanks!

**Reply by albertlarsan68** [reply]
I am using a Samsung Galaxy S20 FE. I will share a screen recording when I come across it again.

**Reply by HenSquared** [reply]
I'll go ahead and open an issue for this with our team! Thanks.

**Reply by albertlarsan68** [reply]
I got a screen recording of one, but it seems GitHub doesn't like it. Do you have another mean of receiving it ?

**Reply by HenSquared** [reply]
Oh hmmm. 🤔 

Cann you upload to a 3rd party video or file hosting platform like youtube? Then share the link with me here.

**Reply by albertlarsan68** [reply]
Here it is:

https://user-images.githubusercontent.com/74931857/211064544-6eef3cc1-b120-4680-911c-5937296eefa7.mp4

Uploading it from a computer works, but not from GitHub Mobile...
This time there was not an infinite loop, but the commends and label assignments are duplicated.
